go/teams/list.go

```diff
@@ -81,13 +83,51 @@ func List(ctx context.Context, g *libkb.GlobalContext, arg keybase1.TeamListArg)
 		}
 	}
 
+	annotatedInvites := make(map[keybase1.TeamInviteID]keybase1.AnnotatedTeamInvite)
+	for teamName := range teamNames {
+		t, err := GetForTeamManagementByStringName(ctx, g, teamName, true)
```

Comment:
yeah I was just looking at this, I'll need to decide whether to do this based on server response. However right now the endpoint for --all doesn't return implicit param, so I need to do that first.